### DISCUSSION TITLE
How to set URL of docker deployment?

### DISCUSSION THREAD

**Discussion by gosi-at** [ngoduykhanh/PowerDNS-Admin#1260] [Q&A] [answered]
My docker deployment works really well, thanks for the hard work you put into this projects!

One thing I cannot figure out: I use nginx to handle SSL and forward traffic to the exposed docker port. So some pages of PowerDNS-Admin link to localhost:port, I wonder how I set that to an URL?

[egal_envvars] didnt shed and light, maybe I have overseen something? Base URL also doenst seem to be inside of web ui settings.

Thanks for helping out!

**Comment by subbink** [comment]
Do you have an example of an URL that points to localhost:port ?

**Comment by gosi-at** [comment, marked as answer]
Thanks for getting back at this, I think I messed up my nginx config, I played a bit and with those parameters to the proxy pass it started to work:

  proxy_set_header 		    Host $host:$server_port;
  proxy_set_header 		    X-Forwarded-Proto $scheme;
  proxy_set_header                  X-Real-IP $remote_addr;
  proxy_set_header                  X-Forwarded-For $proxy_add_x_forwarded_for;

So the whole vhost now looks like this:

server {
  listen 443 ssl http2;
  server_name powerdns.example.com;

  include snippets/ssl.conf;

  ssl_certificate /etc/acme/certs/powerdns.example.com/fullchain.pem;
  ssl_certificate_key /etc/acme/certs/powerdns.example.com/key.pem;

  access_log                /var/log/nginx/powerdns.example.com.access.log combined;
  error_log                 /var/log/nginx/powerdns.example.com.error.log;

  client_max_body_size              10m;
  client_body_buffer_size           128k;
  proxy_redirect                    off;
  proxy_connect_timeout             90;
  proxy_send_timeout                90;
  proxy_read_timeout                90;
  proxy_buffers                     32 4k;
  proxy_buffer_size                 8k;
  proxy_set_header 		    Host $host:$server_port;
  proxy_set_header 		    X-Forwarded-Proto $scheme;
  proxy_set_header                  X-Real-IP $remote_addr;
  proxy_set_header                  X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_headers_hash_bucket_size    64;

  location / {
	  proxy_pass http://localhost:9191/;
	  proxy_read_timeout    120;
	  proxy_connect_timeout 120;
	  proxy_redirect        off;
  }
}
